airbyte-integrations/connectors/source-sentry/acceptance-test-config.yml

```diff
@@ -8,6 +8,19 @@ acceptance_tests:
         extra_fields: no
         exact_order: no
         extra_records: yes
+      ignored_fields:
+        project_detail:
+        - name: features
+          bypass_reason: "Order features return randomly"
+        - name: organization/features
+          bypass_reason: "Order features return randomly"
+        - name: plugins/*/features
+          bypass_reason: "Order features return randomly"
+        projects:
+          - name: features
+            bypass_reason: "Order features return randomly"
+          - name: organization/features
+            bypass_reason: "Order features return randomly"
```

Comment:
For the random ordering-type problems.  If it isn't a crazy ask, I think it would be better to sort the record properties ourself rather than skip testing that stream or property.

Reply:
I was create follow up issue https://github.com/airbytehq/airbyte/issues/23860 to unblock this PR.